code-for-canada/project-thundercat
# [Look and Feel] Mini banner on eMIB welcome and howto pages

# Description

Adding the mini banner to eMIB welcome and howto pages, as per look and feel. Welcome page will likely be removed in a later PR, but adding it for consistency.

Also using H1 in place of H2.

## Type of change

Please delete options that are not relevant.

- Code cleanliness or refactor

## Screenshot

Before:


After:
![image](https://user-images.githubusercontent.com/2746350/53102073-ffd20b00-34f8-11e9-8851-c79fb97d87cb.png)

# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to Prototype Page
2. Start eMIB Sample Test
3. Verify that banner is present on Welcome and HowTo page, above header.

Screenshot of unit tests run passing:
![image](https://user-images.githubusercontent.com/2746350/53102226-3f005c00-34f9-11e9-9f53-03b944737053.png)

# Checklist

Applicable for all code changes.
